style.css

```diff
@@ -32,106 +29,122 @@ hr {
 .wrapper{
 	height: 80vh;
 	display: grid;
-	grid-template-rows: 200px 15px 250px 250px 250px 5%;
-	grid-template-columns: 1fr 2fr 2fr 2fr 2fr 1fr; /*Left, Boxes Left x2, Boxes Right x2, Right*/
+	grid-template-rows: 4vw 49vw 12vw 12vw;
+	grid-template-columns: auto;
+ }
+
+.wrapper2{
+	height: 80vh;
+	display: grid;
+	grid-template-rows: 4vw 58vw 12vw;
+	grid-template-columns: auto;
  }
 
  /*----------------BURGER NAV---------------------*/
+ 
+ .nav{
+	box-sizing: border-box;
+	background-color: #f7f7f7;
+	padding-left: 1vw;
+	padding-right: 1vw;
+	background: rgba(204, 204, 204, 0.7);
+	grid-column: 1 / span 6;
+	grid-row: 1;
+}
 
 #menu__toggle {
-	  opacity: 0;
+	opacity: 0;
 }
 
 #menu__toggle:checked + .menu__btn > span {
-	  transform: rotate(45deg);
+	transform: rotate(45deg);
 }
 
 #menu__toggle:checked + .menu__btn > span::before {
-	  top: 0;
-	  transform: rotate(0deg);
+	top: 0;
+	transform: rotate(0deg);
 }
 
 #menu__toggle:checked + .menu__btn > span::after {
-	  top: 0;
-	  transform: rotate(90deg);
+	top: 0;
+	transform: rotate(90deg);
 }
 
 #menu__toggle:checked ~ .menu__box {
-	  left: 0 !important;
+	left: 0 !important;
 }
 
 .menu__btn {
-	  position: fixed;
-	  top: 40px;
-	  left: 20px;
-	  width: 26px;
-	  height: 26px;
-	  cursor: pointer;
-	  z-index: 3;
+	position: absolute;
```

Comment:
I changed 'px' to 'vw' as it makes the design responsive on any desktop screen size